workspaces/ssh.md

```diff
@@ -52,3 +52,15 @@ workspace:
 ```console
 rsync -e "coder sh" -a --progress ~/. my-env:~
 ```
+
+## Forwarding dev URLs
+
+To access your server via SSH port forwarding, [create a dev URL](devurls.md)
+and access the dev URL port using:
+
+```console
+ssh -L [localport]:localhost:[remoteport] coder.[workspace]
+```
```

Comment:
Requires having run `coder config-ssh` with the coder CLI. I think we'll have to call this out as a pre-requisite somewhere